src/main/java/com/trzewik/jdbc/db/Account.java

```diff
@@ -0,0 +1,25 @@
+package com.trzewik.jdbc.db;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Getter
+@EqualsAndHashCode
```

Comment:
jestem ciekawy czy jesteś pewny że dobrze określono ten bardzo ważny combo (equals+hash-code) - wyjaśnij czemu taki plz

Reply:
wcześniej było `@EqualsAndHashCode` na wszystkich polach, ponieważ wszystkie pola były unikalne... ale nie wiem czy to miało sens.
Obecnie zostawiłem na samym `userId` i dopóki nie używam hibernate'a, to powinno być ok. W przypadku gdy zacznę tutaj używać hiberante'a to po dodaniu adnotacji `@Id` hash code dla każdego obiektu będzie musiał być taki sam [url](https://vladmihalcea.com/how-to-implement-equals-and-hashcode-using-the-jpa-entity-identifier/)